### PR TITLE
docker-compose

### PR DESCRIPTION
**Pull request by almerico**

### CODE REVIEW

File: docker-compose/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DOCKER_IP=192.168.1.9
```
Comment (erdemedeiros) [inline docker-compose/.env]: Can we add comments to this file? In this case can you add the comment `Replace by your IP address`

File: docker-compose/Makefile [hunk 2]
```diff
@@ -0,0 +1,74 @@
+SHELL := /bin/bash
```
Comment (erdemedeiros) [inline docker-compose/Makefile:1]: Is it possible to print the URLs to use at the end of the make command? I mean:
- Modeling app: http://192.168.1.9.nip.io/activiti-cloud-modeling
- Runtime bundle entry point: http://192.168.1.9.nip.io/rb-my-app
...

**Review by erdemedeiros — COMMENTED** [review]
Great work @almerico!
See my questions bellow.

File: docker-compose/.env (hunk 1 above)
Comment (almerico) [reply docker-compose/.env]: Ok

File: docker-compose/Makefile (hunk 2 above)
Comment (almerico) [reply docker-compose/Makefile:1]: Can you please add more details what do you mean? In the end of output?

**Comment by almerico** [comment]
There are will be some updates related to acceptance scenarios run. Will let you posted.

File: docker-compose/Makefile (hunk 2 above)
Comment (erdemedeiros) [reply docker-compose/Makefile:1]: yes, when you execute `make all` you have the output: 
```
docker-compose up -d activiti-cloud-audit
Creating activiti-cloud-audit ... done
make activiti-cloud-notifications-graphql/up
docker-compose up -d activiti-cloud-notifications-graphql
Creating activiti-cloud-notifications-graphql ... done
```

I was wondering if it's possible to print the links after this. Similar to what we have when we execute `helm`.

Comment (almerico) [reply docker-compose/Makefile:1]: added updates

**Comment by almerico** [comment]
Fixes https://github.com/Activiti/Activiti/issues/2658

**Pull request merged**